dapr/aio/clients/grpc/client.py

```diff
@@ -1103,6 +1110,298 @@ async def unlock(
         return UnlockResponse(status=UnlockResponseStatus(response.status),
                               headers=await call.initial_metadata())
 
+    async def start_workflow(
+            self,
+            workflow_component: str,
+            workflow_name: str,
+            input: Union[Any, bytes, None] = None,
+            instance_id: Optional[str] = None,
+            workflow_options: Optional[Dict[str, str]] = dict(),
+            send_raw_bytes: bool = False) -> StartWorkflowResponse:
+        """Starts a workflow.
+
+            Args:
+                workflow_component (str): the name of the workflow component
+                                    that will run the workflow. e.g. `dapr`.
+                workflow_name (str): the name of the workflow that will be executed.
+                input (Union[Any, bytes, None]): the input that the workflow will receive.
```

Comment:
Per our discussion, let's make the documentation more explicit
```suggestion
                input (Union[Any, bytes, None]): the input that the workflow will receive.
                                                 The input value will be serialized to JSON
                                                 by default. Use the send_raw_bytes param
                                                 to send unencoded binary input.
```
Please update the docs both for sync and async variants, as well as for the raise_event methods.